solutions/USACO jan18-billboard.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: usaco-759
+source: USACO Bronze 2018 January
+title: Blocked Billboard II
+author: Melody Yu
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_billboard_bronze_jan18.html)
+
+<LanguageSection>
+<JavaSection>
+See official editorial: http://www.usaco.org/current/data/sol_billboard_bronze_jan18.html
```

Comment:
might as well just not include this part

Reply:
got it